### ISSUE TITLE
struct2depth training is super slow

### ISSUE THREAD

**Issue by aj96** [tensorflow/models#6736]
How long is it supposed to train the model? I am training on a Tesla P-100, and it is taking about 150s to train for 100 steps. The default number of training steps is 10 million. At this rate, it will take around 173 days to train?? (10,000,000 steps) / (100 steps / 150 s). I am training on kitti dataset and used the matterport MASK-RCNN to generate the masks.

Is this right?

What is the top-level directory of the model you are using: struct2depth
Have I written custom code: No
OS Platform and Distribution: Ubuntu 18.04
TensorFlow installed from: pip install tensorflow-gpu
TensorFlow version: 1.13
Bazel version: N/A
CUDA/cuDNN version: 10.0 / 7.4.1
GPU model and memory: Tesla P100 16GB
Exact command to reproduce: the command provided to train the model

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ezorfa** [comment]
Hi, Did it indeed take that longer to train this network?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
